direct/src/extensions_native/NodePath_extensions.py

```diff
@@ -168,8 +185,9 @@ def pPrintString(self, other = None):
         """
         Deprecated.  pretty print
         """
-        print("NodePath.pPrintString() is deprecated.")
+        print("")
```

Comment:
There doesn't need to be an empty print statement here

Reply:
Removed

Reply:
Deleted